README.md

```diff
@@ -136,8 +157,10 @@ This repository hosts an e-commerce project tailored for baby products. It is de
    ```bash
    docker run -d --restart on-failure -p 8025:8025 \
    -v /baby-tools-shop/project_images:/app/project_images \
-   -v /baby-tools-shop/.env:/app/.env \
+   -v /babyshop_db:/app/db \
+   -v /babyshop_app/.env:/app/.env \
```

Comment:
Bind environment file using docker explizit argument for that. See this link https://stackoverflow.com/questions/30494050/how-do-i-pass-environment-variables-to-docker-containers for more info.

Example: docker run --env-file ./env.list ubuntu bash